python/cudf/cudf/bindings/rolling.pyx

```diff
@@ -62,6 +62,8 @@ def apply_rolling(inp, window, min_periods, center, op):
     else:
         if isinstance(op, tuple):
             cpp_str = op[0].encode('UTF-8')
+            if op[1] not in dtypes:
+                raise TypeError("Type '%s' is not supported in cuDF", op[1])
```

Comment:
Let's be more clear; how about something like `TypeError("Result of window function has unsupported dtype {}"format(op[1]))`

Also, to do string formatting in the way you want, you'll have to write:

```
"Type '%s' is not supported in cuDF" % op[1]
```

Reply:
Yeah I was using the C++ syntax ... but it should be `TypeError("Result of window function has unsupported dtype {}".format(op[1]))` right? (notice the dot before `format`)